scripts/install

```diff
@@ -12,7 +12,7 @@ if [ -z "$TAG_PROTOTYPE_CJDNS_PI" ]; then
     TAG_PROTOTYPE_CJDNS_PI=master
 fi
 
-if ps xa | awk '{print $5}' | grep dpkg | grep -q grep; then
+if ps xa | awk '{print $5}' | grep -q dpkg; then
```

Comment:
Will this miss `sudo dpkg ...`?

Either way I think this line can be replaced with `pgrep -axq dpkg`